packages/thumbprint-react/components/Image/index.tsx

```diff
@@ -44,12 +107,12 @@ const Image = forwardRef((props, outerRef) => {
     // Used by srcSet to determine which image in the list will be requested. This value has to be
     // calculated client-side because we don't know the viewport width.
 
-    const computeSizes = () =>
+    const computeSizes = (): string =>
         containerRef && containerRef.clientWidth ? `${containerRef.clientWidth}px` : '0px';
 
     // If `forceEarlyRender` is supplied use that value, otherwise use the computed width.
 
-    const sizes = forceEarlyRender || computeSizes();
+    const sizes = forceEarlyRender ? computeSizes() : '';
```

Comment:
Is this supposed to be:

```suggestion
    const sizes = forceEarlyRender ? forceEarlyRender : computeSizes();
```

Reply:
The issue here may be that `forceEarlyRender` can be either a boolean or a string. 

In the old code, if `forceEarlyRender` were `true`, the `sizes` attribute would be equal to `true`, not a number. This is invalid, of course, so TypeScript complains.

@tomgenoni — Should `forceEarlyRender` have been `PropTypes.oneOfType([PropTypes.bool, PropTypes.string])`? Or just `PropTypes.string`?

Reply:
ah yeah, i've misunderstood this. i'm assuming `forceEarlyRender` should always be a string, and if it's a truthy string you use that, otherwise you use `computeSizes`.